src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -33,4 +39,106 @@ public void getTypeEqualsHouse() {
         assertThat(structure.getType()).isEqualTo(StructureType.HOUSE);
     }
 
+    @Test
+    public void deliverItemTest() {
+        // Arrange
+        final House house = new House(new Position());
+        final IItem item = new IItem() {
+        };
+
+        // Act
+
+        // Assert
+        assertThat(house.deliverItem(item)).isTrue();
+    }
+
+    @Test
+    public void builtStatusTest() {
+        // Arrange
+        final House house = new House(new Position());
+        final IItem item = new IItem() {
+        };
+
+        // Act
+        house.deliverItem(item);
+
+        // Assert
+        assertThat(house.builtStatus()).isEqualTo(0.01f);
```

Comment:
There's no way for the test to know what the status will go up by 0.01f. How does it know? Because you know how its implemented in the actual class. This means the test is relying on implementation details, which it shouldn't. Instead, you can just check that the status goes up.